tests/sync/test_sync.py

```diff
@@ -1,17 +1,18 @@
 import unittest
+import pytz
+from ado_asana_sync.sync.sync import *
 from asana import UserResponse
-from ado_asana_sync.sync.sync import work_item, get_task_user_email, matching_user
 from azure.devops.v7_0.work_item_tracking.models import WorkItem
+from datetime import datetime, timezone
 
 
 class TestWorkItem(unittest.TestCase):
     def setUp(self) -> None:
-        self.test_item = work_item(
+        self.test_item = TaskItem(
             ado_id=123,
+            ado_rev=3,
             title="Test Title",
             item_type="User Story",
-            status="Test Status",
-            description="Test Description",
             url="https://testurl.example",
```

Comment:
**Suggestion:** In the test cases where TaskItem objects are created, consider adding assertions to check if the objects are created correctly. This will help ensure that the TaskItem constructor is working as expected.
```suggestion
        self.test_item = TaskItem(
            ado_id=123,
            ado_rev=3,
            title="Test Title",
            item_type="User Story",
            url="https://testurl.example",
        )

        # Assert that the TaskItem object is created correctly
        self.assertEqual(self.test_item.ado_id, 123)
        self.assertEqual(self.test_item.ado_rev, 3)
        self.assertEqual(self.test_item.title, "Test Title")
        self.assertEqual(self.test_item.item_type, "User Story")
        self.assertEqual(self.test_item.url, "https://testurl.example")
```